### ISSUE TITLE
Line-height should be 1.5 to comply the WCAG guideline

### ISSUE THREAD

**Issue by NagayamaToshiaki** [twbs/bootstrap#34629]
This css's base line-height is 1.15 and this violates the WCAG Success Criterion 1.4.12: Text Spacing.
To suits the criterion, line-height must be "at least 1.5 times the font size".
So, html element's style should be:
html {
  font-family: sans-serif;
  line-height: 1.5;
  -webkit-text-size-adjust: 100%;
  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
}

**Comment by ffoodd** [comment]
That's not [what WCAG 1.4.12 says](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html). It stipulates that no content should be lost nor occurred when line-height is set to 1.5 (among other things, BTW).

It does not require it to be the default.

Thanks for opening issue but we're safe regarding WCAG (at least for this criteria!).
